pkgs/development/python-modules/cssutils/default.nix

```diff
@@ -1,29 +1,43 @@
 { lib
 , buildPythonPackage
-, isPy27
+, pythonOlder
 , fetchPypi
+, setuptools-scm
+, toml
+, importlib-metadata
 , mock
 , pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "cssutils";
-  version = "2.0.0";
+  version = "2.2.0";
 
-  disabled = isPy27;
+  disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "984b5dbe3a2a0483d7cb131609a17f4cbaa34dda306c419924858a88588fed7c";
+    sha256 = "5bef59f6b59bdccbea8e36cb292d2be1b6be1b485fc4a9f5886616f19eb31aaf";
   };
 
+  nativeBuildInputs = [
+    setuptools-scm
+    toml
```

Comment:
toml can be removed here since [`4b7e688` (#115349)](https://github.com/NixOS/nixpkgs/pull/115349/commits/4b7e688d10da67ac786c78f070fbd13f482a32a3) which is currently only in staging.

Reply:
Well this isn't targetting staging.

Reply:
I am not sure if this would cause a duplicated input or another error when staging ist merged.

Reply:
No, that's not a problem.